COLLABORATOR_GUIDE.md

```diff
@@ -229,6 +229,10 @@ is designed to allow validation of changes to the copy of V8 in the Node.js
 tree by running the standard V8 tests. It should be run whenever the
 level of V8 within Node.js is updated or new patches are floated on V8.
 
+* [`node-test-commit-custom-suites`](https://ci.nodejs.org/job/node-test-commit-custom-suites/)
+runs tests that are in the `test/internet` directory. Those are not run by the
+other jobs listed here.
```

Comment:
While it is true that the default `CI_JS_SUITES` parameters to the job run `internet` the job is intended to run any test(suite) with custom arguments (I think you could even simulate the stress job by passing `--repeat` to `TEST_ARGS`).

Reply:
I was trying to provide the minimum information to be useful. (I'd like to see `pummel` added FWIW.) If you have some alternate or additional text, do feel free to add it!